code/playerman/playercontrol.cpp

```diff
@@ -1608,24 +1606,34 @@ bool player_inspect_cargo(float frametime, char *outstr)
 		vm_vec_normalized_dir(&vec_to_cargo, &cargo_objp->pos, &Player_obj->pos);
 		float dot = vm_vec_dot(&vec_to_cargo, &Player_obj->orient.vec.fvec);
 		if ( dot < CARGO_MIN_DOT_TO_REVEAL ) {
-			if (reveal_cargo)
-				strcpy(outstr,XSTR( "cargo: <unknown>", 86));
-			else
-				strcpy(outstr,XSTR( "not scanned", 87));
-			hud_targetbox_end_flash(TBOX_FLASH_CARGO);
-			Player->cargo_inspect_time = 0;
-			return true;
+			if (reveal_cargo) {
+				if (cargo_sp->cargo_title[0] != '\0') {
```

Comment:
#5922 asked for the ability to hide the `cargo:` prefix entirely. Could we support that as well? I hate to suggest overloading the hash sign yet again, but it is something of a convention.

Reply:
Ah, good thought. I'll come up with something for that.

Reply:
Title starting with '#' will now make the Title entirely invisible.